digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -128,7 +129,7 @@ public CommandStatus run(
             }
         }
         catch (ConfigException e) {
-            logger.debug("Fall back to DockerCommandExecutor: {} {}", e.getMessage(), e.getCause() != null ? e.getCause().getMessage() : "");
+            logger.warn("Fall back to DockerCommandExecutor: {} {}", e.getMessage(), e.getCause() != null ? e.getCause().getMessage() : "");
```

Comment:
I don't think it is warning.
ECS CommandExecutor is default and fall back to Docker is current behavior.

Reply:
Yes, and it should be warning and an exceptional case.

Reply:
This class is ECSCommandExecutor not DockerCommandExecutor.

Reply:
I'm not familiar with the background of the behavior, but Digdag users have no way to specify the type of CommandExecutor and EcsCommandExecutor is always tried at first? https://github.com/treasure-data/digdag/blob/c8e3b8e17c74f2535fc8677fc61e1af70b8718d9/digdag-standards/src/main/java/io/digdag/standards/command/CommandExecutorModule.java#L16

If so, `debug` sounds good to me.

Reply:
actually, need to disable local docker execution considering security risks at least in TD.
Some debug-level logs existence in TD production was surprising.

Reply:
Changed for OSS version.
https://github.com/treasure-data/digdag/pull/1603/commits/15e0dcdc342d109b60d0aa39bd5f7a9cd90dfd3d

TD version should output error-level logs and throw an exception considering security risks.

Reply:
BTW, we can find local docker execution unexpectedly in TD production logs. It should not be allowed.

Reply:
Yes. as @komamitsu point out, EcsCommandExecutor is always used and fall back to Docker -> Simple.
@myui I think you add a method to override in TD internal and it is good way if you want to change the behavior for TD specific.